packages/perseus-core/src/widgets/core-widget-registry.ts

```diff
@@ -71,6 +73,25 @@ export function getDefaultWidgetOptions(type: string) {
     return widgetLogic?.defaultWidgetOptions || {};
 }
 
+export function isAccessible(
```

Comment:
Just a reminder that we should delete the same-named function in `@khanacademy/perseus` so we don't have two (we discussed this in today's sync, but wanted to note it here also). :)